### DISCUSSION TITLE
Calendar Creation

### DISCUSSION THREAD

**Discussion by hardikcelinel** [spatie/icalendar-generator#92] [General]
Hi,

I read the documentation for the calendar. but I would like to know that, does it automatically create calendar without doing anything in google calendar or outlook calendar or any calendar?

The thing is that, I used the create calendar code and used the create event code and display as string in browser. but it doesn't show up in google calendar or anyother calendar. So do I need to do any further steps?

Kindly help.

Thanks.

**Comment by rubenvanassche** [comment]
Take a look at this: https://github.com/spatie/icalendar-generator#use-with-laravel
